### ISSUE TITLE
Using XbimEssentials in a C++ environnement?

### ISSUE THREAD

**Issue by pierrenavarra** [xBimTeam/XbimEssentials#370]
Hi,
Last days I made a simple application in C# to get all materials of a specific wall. Everything was ok.
A get the XbimEssential NuGet package and I was able to code : using Xbim.Ifc;
Now, I'd like to do the same thing with a C++ application.
Is it possible? How to get packages?
Thx

**Comment by martin1cerny** [comment]
Yes, this should be possible. But this is not specific to Xbim. You should be able to use any method for C#/C++ interoperability. If you are using [/CLI](https://en.wikipedia.org/wiki/C%2B%2B/CLI), you can just reference C# code directly.

**Comment by pierrenavarra** [comment]
Hi martin.
Whow! I'm not sure to understand this solution....

**Comment by martin1cerny** [comment]
As I said, XbimEssentials is no different to any other .NET NuGet package. You would be better to search official Microsoft documentation or StackOverflow (or ask there) to find more about how to use NuGet packages in C++/Unreal projects.
